src/main/webapp/assets/js/analytics.js

```diff
@@ -0,0 +1,80 @@
+$(document).ready(function(){
```

Comment:
Generally:
Mixed tabs and spaces
Mixed  single and double quotes
Might be good to provide a JSHint header at the top of the file to help with validation like:
'use strict';
/* jshint  strict: true*/
/* global $*/